uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlements.component.spec.js

```diff
@@ -118,6 +119,17 @@ define([
       });
     });
 
+    describe('when user wants to see a comment to an entitlement', function () {
```

Comment:
The test is confusing because it implies that an error will be displayed, even if it's not. `notification.error` is being used because it has the same aesthetics as the message that is to be displayed to the user, but we should make a difference even if it's only code-wise.

A `notification.notice` should be added and used instead. This will require refactoring the notifications service a bit.